src/backend/wine/winecommand.py

```diff
@@ -506,10 +507,43 @@ def __get_gamescope_cmd(self, return_steam_cmd: bool = False) -> str:
 
         return " ".join(gamescope_cmd)
 
+    def vmtouch_preload(self):
+        vmtouch_command = "/app/bin/vmtouch"
```

Comment:
I think shutil.which("vmtouch") better fit the case as we can also check it before executing. Also moving to globals.py like other checks, e.g.:
https://github.com/bottlesdevs/Bottles/blob/3ef4024fa2213abba405485aa5ac07fcff4c63b1/src/backend/globals.py#L95
and then check:
https://github.com/bottlesdevs/Bottles/blob/3ef4024fa2213abba405485aa5ac07fcff4c63b1/src/backend/wine/winecommand.py#L430